src/GitTfs.VsCommon/TfsHelper.Common.cs

```diff
@@ -1358,6 +1358,11 @@ public bool IsExistingInTfs(string path)
             return VersionControl.ServerItemExists(path, VersionSpec.Latest, DeletedState.Any, ItemType.Any);
         }
 
+        public bool ValidateTfsFolder(string remoteTfsPath)
```

Comment:
If we keep this  (and I am not convinced we need to, see comments further down), then I'd prefere a name like `IsFolderExistingInTfs`, similar to the method `IsExistingInTfs` above

Reply:
Certainly fine with that change or any change that still allows me to clone folders. :)